development/views_api.py

```diff
@@ -6,15 +6,17 @@
 
 @csrf_exempt
 def match_list(request):
-    # if request.method == 'GET':
-    # matchs = Match.objects.all()
-    # serializer = MatchSerializer(matchs, many=True)
-    # return JsonResponse(serializer.data, safe=False)
-
     if request.method == 'POST':
-        data = JSONParser().parse(request)
+        req_data = JSONParser().parse(request)
+        data = {}
+        try:
+            data['game_id'] = req_data["game_id"]
+            for i, (name, score) in enumerate(req_data["data"], 1):
+                data[f'bot_{i}'] = name
+                data[f'score_p_{i}'] = score
```

Comment:
could you move this behavior to a single method/function/util in order to test it separately?

Reply:
done in convert_data method